server/zanata-war/src/main/webapp/project/project.xhtml

```diff
@@ -231,6 +231,9 @@
         </li>
       </ui:fragment>
       <li>
+        <ui:fragment
+            rendered="#{not empty projectHome.instance.homeContent or
+            identity.hasPermission(projectHome.instance, 'update')}">
```

Comment:
This might be a question for Kathryn, but since the edit section is in a different tab, I think it's pretty self-explanatory that there is no About content.
I see you already made the changes, so based solely on my own opinion I will approve this. Feel free to contact Kathryn if you want to UX related advice.